### PR TITLE
Support `…` as punctuation in `list-item` rule

### LINKED ISSUE #97: … (which is not ...) could be considered valid punctuation.
I get a 

```
List item description must end with proper punctuation                          remark-lint:awesome-list-item
```
On a line that finishes with … (the utf-8 caracter, not three dots `...`).

### PR DESCRIPTION
**Pull request by arthurlutz**
Closes #97

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you add a test?

File: rules/list-item.js [hunk 1]
```diff
@@ -276,7 +276,7 @@ function validateListItemPrefix(descriptionText, prefixText) {
 }
 
 function validateListItemSuffix(descriptionText, suffixText) {
-	if (/[.!?]\s*$/.test(suffixText)) {
+	if (/[.!?…]\s*$/.test(suffixText)) {
 		// Description ends with '.', '!', or '?'
```
Comment (sindresorhus) [inline rules/list-item.js:280]: You need to update this comment too.

Comment (sindresorhus) [reply rules/list-item.js:280]: And probably https://github.com/sindresorhus/awesome-lint/pull/98/files#diff-6e9aa7b10a8503bfb11fbb683cfe854bR284 too.

**Comment by sindresorhus** [comment]
Bump :)

File: rules/list-item.js (hunk 1 above)
Comment (arthurlutz) [inline rules/list-item.js:280]:
```suggestion
		// Description ends with '.', '!', '?' or '…'
```

**Comment by arthurlutz** [comment]
Fixed in the documentation comment, not sure what your second link is pointing too though.

**Comment by sindresorhus** [comment]
> Can you add a test?

⬆️

**Comment by sindresorhus** [comment]
@arthurlutz Bump

**Pull request merged**